edit_course/operations/configure.py

```diff
@@ -276,7 +298,7 @@ def configure_content(instance, url):
         if not "name" in c:
             errors.append(_("Category requires a name."))
             continue
-        category, flag = instance.categories.get_or_create(name=str(c["name"]))
+        category, flag = instance.categories.get_or_create(name=format_localization(c["name"]))
```

Comment:
This requires more looking into. Technically categories should have `key/slug` and `display string` instead of them both being the same field. We need to look into this a bit more to see if this works as expected.